core-bundle/src/EventListener/DataContainer/ContentCompositionListener.php

```diff
@@ -95,7 +95,7 @@ public function __construct(ContaoFramework $framework, Security $security, Page
      */
     public function renderPageArticlesOperation(array $row, ?string $href, string $label, string $title, ?string $icon): string
     {
-        if (!$this->security->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_MODULE, 'article')) {
+        if (null === $href && null === $icon || !$this->security->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_MODULE, 'article')) {
```

Comment:
```suggestion
        if ((null === $href && null === $icon) || !$this->security->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_MODULE, 'article')) {
```

<img width="905" alt="" src="https://user-images.githubusercontent.com/1192057/109843615-8aeb4d00-7c4b-11eb-9d16-859f4e76461e.png">